### ISSUE TITLE
Add the ability to "skip vision decoder" to make it easier to support future models

### ISSUE THREAD

**Issue by vYLQs6** [ollama/ollama#8097]
Since most AI companies have their own vision models, in 2025, there might be more models that release only a vision variant instead of both text and vision variants.

To make it easier to support these new models, it would be nice to be able to skip the vision decoder and infer only the text part. 

Is this possible? It would certainly make it much easier and quicker to support new models.

**Comment by ghost-2362003** [comment]
so if i understand correctly this feature would give the user an option to whether skip or use the vision decoder and use the text inference part only, considering the trend of the AI companies in  2025

feel free to correct me if needed

**Comment by pdevine** [comment]
I don't think this is something we'd do at this time. If we only supported the text portion (which is usually released separately anyway like in the case of llama3) we'd get a ton of issues asking when the vision part was going to be released. I realize it's frustrating at this point (w/ qwen2.5-vl and pixtral not released), but we are making a lot of progress on the new engine and will be able to support multi-modal models much more quickly in the future.

If we only supported half of the model that would just slow us down more. Thank you for the suggestion though.
